src/NativeScript/inspector/CachedResource.mm

```diff
@@ -2,26 +2,50 @@
 #include "MimeTypeHelper.h"
 
 namespace Inspector {
+Inspector::CachedResource createCacheResource(WTF::HashMap<WTF::String, Inspector::CachedResource>& cachedResources, NSString* basePath, NSString* filePath, NSString* prefix) {
```

Comment:
Can these functions be made static? Also, the `cachedResources` argument seems unused.